src/index.js

```diff
@@ -1,3 +1,5 @@
 import SupcommunityScraper from './api/index';
 
-export default SupcommunityScraper;
\ No newline at end of file
+export { SupcommunityScraper };
+export default SupcommunityScraper;
+module.exports = SupcommunityScraper;
```

Comment:
This won't work. If you want to achieve both correctly, make "default" be a static property of the class, and export it whole via `export = SupcommunityScraper`, but do try it beforehand.